shared/tags.js

```diff
@@ -0,0 +1,21 @@
+// Library for handling tags
+'use strict';
+var _ = require('lodash');
+
+// Tags are string comma separated in the front end
+var toArray = function(tags) {
+  if (typeof tags === 'string') {
```

Comment:
So if it's not a string, we just return `undefined`? Won't that cause problems harder to debug than `object doesn't have method 'trim'`?